packages/financial-templates-lib/src/helpers/GasEstimator.ts

```diff
@@ -156,8 +156,9 @@ export class GasEstimator {
   // network then you will pay the prevailing base fee + the max priority fee. if not london then pay the latest fast
   // gas price.
   getExpectedCumulativeGasPrice(): number {
-    if (this.type == NetworkType.London) return this.latestBaseFee + this.latestMaxPriorityFeePerGasGwei;
-    else return this.lastFastPriceGwei;
+    if (this.type == NetworkType.London)
+      return this.latestBaseFeeGwei * 1e9 + this.latestMaxPriorityFeePerGasGwei * 1e9;
```

Comment:
as we are storing this as gwei, we need to scale it back to wei when returning.

note that there was actually a bug in this before where the `latestMaxPriorityFeePerGasWei` was not scaled by 1e9 meaning that we were effectively not considering the maxfeepergas in the cumulative gas price.